test/acceptance/team_test.rb

```diff
@@ -37,4 +37,32 @@ def test_managing_a_team_as_sole_manager
       end
     end
   end
+
+  def test_team_pagination
+
+    100.times do |index|
```

Comment:
We realized it was about the activity of the team not the amount of members that triggers the page to paginate. So we really could have five members with a lot of activity.

Our question at this point is how do you add activity to a user/ream member?
We poked around at the end of the night but didn't find anything definitive yet.